kinder/pkg/cluster/manager/actions/copy-certs.go

```diff
@@ -37,23 +39,62 @@ func CopyCertificates(c *status.Cluster) error {
 	return nil
 }
 
-// copyCertificatesToNode automate copy of certificates from the bootstrap control-plane node to the target node
+// copyCertificatesToNode copies certificate files from the bootstrap node to another node
 func copyCertificatesToNode(c *status.Cluster, n *status.Node) error {
 	// define the list of necessary cluster certificates
+	const (
+		keyCA           = "ca.key"
+		keyCAFrontProxy = "front-proxy-ca.key"
+		keyCAEtcd       = "etcd/ca.key"
+	)
```

Comment:
Wondering why introducing const for keys, given that they are used in a single point down in the code.
Also, why not const for the crt?

Reply:
these are used in a couple of places, but i've changed them to inline.